ci-operator/step-registry/upi/conf/vsphere/clusterbot/pre/upi-conf-vsphere-clusterbot-pre-commands.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cluster_name=${NAMESPACE}-${JOB_NAME_HASH}
+base_domain="${cluster_name}.vmc-ci.devcluster.openshift.com"
+
+export AWS_DEFAULT_REGION=us-west-2  # TODO: Derive this?
```

Comment:
For basic AWS jobs, we [get the region from the Boskos lease][1].  Would that work here?  Or are we just using AWS to host the load balancer, where quotas and API throttling and whatnot is spaced out enough that we don't need a covering Boskos lease?  Looks like there is a way [to request multiple leases][2], if we wanted to track that, although it would probably involve knock-on effects as existing AWS jobs also needed to explicitly lease one LB / Route53-API-request-load unit.

[1]: https://github.com/openshift/release/blob/c2cc23ec9c1f77c3777df1c55be0cf8c8f3ce863/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L18
[2]: https://docs.ci.openshift.org/docs/architecture/step-registry/#explicit-lease-configuration

Reply:
Thanks for taking a look at that.  We are just using AWS for the load balancer in this case and for vSphere jobs, the LEASED_RESOURCE doesn't contain the region since vSphere obfuscates the underlying AWS infra.  It will always be on us-west-2.